src/ace/adm_set.py

```diff
@@ -259,6 +260,10 @@ def load_from_dirs(self, dir_paths:List[str]) -> int:
         :return: The number of ADMs read from that directory.
         '''
         LOGGER.debug('Loading from directories %s', dir_paths)
+        # workaround misuse
```

Comment:
**Question**: Is it possible/desirable to fix the misuse at the caller rather than here in this function?

Reply:
Yes I can just change the type signature to allow it normally.